hack/release.sh

```diff
@@ -293,30 +293,8 @@ EOF
 	# Upload repo
 	s3cmd --acl-public "$s3Headers" sync "$APTDIR/" "s3://$BUCKET/ubuntu/"
 	cat <<EOF | write_to_s3 s3://$BUCKET/ubuntu/index
-echo "# WARNING! This script is not going to install aufs on your"
-echo "# system. If you want to use aufs please check the script"
+echo "# WARNING! This script is depreciated. Please use the script"
```

Comment:
spelling is hard hahaha

On Thu, Jul 23, 2015 at 2:44 PM, Tianon Gravi <notifications@github.com>
wrote:

> In hack/release.sh
> <https://github.com/docker/docker/pull/14920#discussion_r35375135>:
>
> > @@ -293,30 +293,8 @@ EOF
> >  	# Upload repo
> >  	s3cmd --acl-public "$s3Headers" sync "$APTDIR/" "s3://$BUCKET/ubuntu/"
> >  	cat <<EOF | write_to_s3 s3://$BUCKET/ubuntu/index
> > -echo "# WARNING! This script is not going to install aufs on your"
> > -echo "# system. If you want to use aufs please check the script"
> > +echo "# WARNING! This script is depreciated. Please use the script"
>
> I think you meant deprecated here. [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/14920/files#r35375135>.
>